Src/library/docking/implementation/configuration/sd_open_config_mediator.e

```diff
@@ -421,14 +418,15 @@ feature {NONE} -- Implementation
 						l_left_zones := internal_docking_manager.query.all_zones_in_widget (l_first)
 					else
 						check False end -- Implied by docking widget must not has empty child
+						create l_left_zones.make (0)
 					end
 
 					if attached l_top_split_area.second as l_second then
 						l_right_zones := internal_docking_manager.query.all_zones_in_widget (l_second)
 					else
 						check False end -- Implied by docking widget must not has empty child
+						create l_right_zones.make (0)
 					end
-					check l_left_zones /= Void and l_right_zones /= Void end -- Implied by previous if clauses
 					if is_all_tools (l_left_zones) or  is_all_tools (l_right_zones) then
```

Comment:
Couldn't we make `is_all_tools` return false when argument is Void? That would save us from having to create them when something goes wrong.